### ISSUE TITLE
Exc. Checksum Error - Mister AO486 Core - Some Titles 

### ISSUE THREAD

**Issue by convolutionbbs** [dwhinham/mt32-pi#300]
Hello,
I have a PI4 2GB model with legacy pixels MT-32 PI HAT XL.  I am using the most recent release 0.21.1 of the MT32-PI software.  I have a new Mister installation at it's most recent revision as well.

I am using this config file from Legacy Pixels:
https://www.legacypixels.com/docs/mt32xl_12_1/mt32-pi.txt

My first test of SQ3 and PQ2 worked as expected.  However, as I went further in to my library, titles such as QFG2 and KQ6CD (among others) return an "Exc. Checksum error" on the on-screen LCD display.  It feels like all the instruments are not loaded correctly and the sound plays some correct instruments but mostly muffled garbage.  KQ6CD plays the speech audio fine as well - but the midi is obviously broken.

I couldn't find much in the way of this error.  Perhaps the communication between the Mister and the PI are set too fast and the MT-32 pi can't keep up?  It's not clear if this is an issue with MT32-PI, AO486, or my setup somehow.

https://www.vogons.org/viewtopic.php?t=8163&p=53326
https://bugs.scummvm.org/ticket/1261

Any help would be appreciated - thanks!

**Comment by dwhinham** [comment]
> Perhaps the communication between the Mister and the PI are set too fast and the MT-32 pi can't keep up?

Nope. The communications happen at the 1980s MIDI standard of 31250bps, and the Pi is far more than capable of "keeping up". It is not possible to send data "too fast" from the MiSTer. If this were the case, you would be seeing buffer overflow errors, not checksum errors. Checksum errors mean invalid data or data corruption.

What *is* possible is that the game is timing-sensitive, so you could try slowing AO486 down. But also, checking the [wiki page](https://en.wikipedia.org/wiki/List_of_MT-32-compatible_computer_games), we see this:

![image](https://user-images.githubusercontent.com/5890747/186196542-15d89ddb-097b-4430-a963-8fed5deb18aa.png)
![image](https://user-images.githubusercontent.com/5890747/186196692-c9c6797b-d23a-4cae-b797-4dd118df5678.png)

...which implies that these games should only be used with an "old" ROM set (e.g. v1.07) and Kings Quest 6 should really be used in SoundFont mode as it's an SC-55 soundtrack.

>It's not clear if this is an issue with MT32-PI, AO486, or my setup somehow.

Most likely the game and/or AO486, especially if other games work fine. These issues can not be fixed from the mt32-pi side.

I am transferring this issue to a discussion in case people who are more knowledgable about these games can help, but you would be better served by a thread on VOGONS or a similar forum, as I know nothing about these games.

FYI, I normally close and ignore issues where people deliberately delete the issue template. Please do not do this in future.
